tests/skibidi-orm/migration_engine/constraints/test_constraints.py

```diff
@@ -0,0 +1,22 @@
+import skibidi_orm.migration_engine.adapters.database_objects.constraints as c
+
+
+def test_ColumnSpecificConstraint_sortable():
+    sorted(
+        [
+            c.PrimaryKeyConstraint(
+                table_name="table_name",
+                column_name="column_name",
+            ),
+            c.NotNullConstraint(
+                table_name="table_name",
+                column_name="column_name",
+            ),
+            c.CheckConstraint(
+                table_name="table_name",
+                column_name="column_name",
+                condition="condition",
+            ),
+        ]
+    )
+    assert True
```

Comment:
maybe it's better to test the order of the sorted constraints, even if it doesn't really matter, than assert true?

Reply:
If order doesn't matter then there is no correct way of sorting them, so tests would be totally artificial. However to improve this test I aded pytest.fail whenever sorted() were to throw an Error indicating, that the list is not sortable